app/domain/authentication/authenticate.rb

```diff
@@ -45,11 +41,11 @@ def authenticator
     end
 
     def validate_authenticator_exists
-      raise Err::AuthenticatorNotFound, authenticator_name unless authenticator
+      raise Errors::Authentication::AuthenticatorNotFound, authenticator_name unless authenticator
```

Comment:
I like this change, but I don't like these long lines.  How about:

```ruby
      raise(
        Errors::Authentication::AuthenticatorNotFound,
        authenticator_name
      ) unless authenticator
```
whenever the long gets too long?

Reply:
sure, i'll do it.